modules/tribes/client/components/TribesList.component.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styled from 'styled-components';
+
+import Tribe from './Tribe';
+import SuggestTribe from './SuggestTribe';
+
+const List = styled.ul`
+  margin: -5px -5px 15px -5px;
+  line-height: 0;
+`;
+
+const Item = styled.li`
+  line-height: 1.42857143; // set back the line-height set by bootstrap
+  padding: 5px;
+  display: inline-block;
+  vertical-align: bottom;
+  // background-color: red;
+  @media (max-width: 615.9999px) {
+    width: 100%;
+  }
+
+  @media (min-width: 616px) and (max-width: 991.9999px) {
```

Comment:
Looking at values like `991.9999px` makes me feel like it should come from a centralized variable somewhere.

Would it make sense to combine these styles with something like [`react-responsive`](https://www.npmjs.com/package/react-responsive)?

Reply:
react-response looks like it's for selectively rendering whole components based on media queries, but here it's about setting css based on media queries.

As styled-components are just strings could use something similar to https://jsramblings.com/2018/02/04/styled-components-media-queries.html.

https://github.com/morajabi/styled-media-query is another approach, but doesn't seem much improvement on the other approach, and needs an extra library.

Reply:
I just found we don't have to specify this fractional `max-width` because the css will care about the latest rule only.

So we could just do
```css
/* @media (min-width: 0) is just for clarity that width values are overwritten by the next one */
/* it can be removed. preferences? */
@media (min-width: 0) {
  width: 100%;
}
@media (min-width: 616px) {
  width: 50%;
}
@media (min-width: 992px) {
  width: 33.3333%;
}
```

When it comes to these breakpoints, they are pretty arbitrary (just taken by observing behavior of angular-grid). The real device breakpoints will [likely be different](https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/). I would prefer not to put the breakpoints here to outside variables, as they're most likely not going to be reused anywhere.

Thank you for the linked articles, I've learned from them and the knowledge will be useful later.

What do you think? Does it make sense?

Reply:
> `/* @media (min-width: 0) is just for clarity that width values are overwritten by the next one */`

My preference would be to not include it, since overriding with media queries is so typical in CSS. So;

```css
width: 100%;

@media (min-width: 616px) {
  width: 50%;
}
```

Reply:
I removed the `max-width` parts of `@media` in 9fcdd0ab1e89d508057425b433571a1394c42565. It works because width is always overwritten by the next media query.
Please give a feedback, especially:
- Does `@media (min-width: 0)` have any clarity benefit, or should we remove it altogether?
- Can we leave the breakpoints hardcoded?

Reply:
I'd prefer not `min-width:0` and it's fine to leave them hardcoded. 👍

Reply:
removed min-width and added some comments in 8f13a627b6f84a5f8af89177045725